src/types.ts

```diff
@@ -0,0 +1,5 @@
+export type ArrayTwoOrMore<T> = [T, T] & T[];
+
+export function isArrayTwoOrMore<T>(value: T[]): value is ArrayTwoOrMore<T> {
```

Comment:
Please add some documentation. Maybe a link to typeguards explanation could help, too.

Reply:
Done.